addons/binding/org.openhab.binding.boschindego/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="boschindego"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <thing-type id="indego">
+        <label>Bosch Indego</label>
+        <description>Indego which supports the connect feature.</description> 
+        <channels>
+            <channel id="state" typeId="state"/>
+            <channel id="textualstate" typeId="textualstate"/>
+            <channel id="errorcode" typeId="errorcode"/>
+            <channel id="statecode" typeId="statecode"/>
+            <channel id="mowed" typeId="mowed"/>
+            <channel id="ready" typeId="ready"/>
+        </channels>
+        <config-description>
+        <parameter name="username" type="text" required="true">
+            <label>Username</label>
+            <description>Username for the Bosch Indego account.</description>
+        </parameter>
+        <parameter name="password" type="text" required="true">
+            <label>Password</label>
+            <description>Password for the Bosch Indego account.</description>
+        </parameter>
+        <parameter name="refresh" type="integer" min="60">
+            <label>Refresh interval</label>
+            <description>Specifies the refresh interval in seconds.</description>
+            <default>180</default>
+        </parameter>
+    </config-description>
+    </thing-type>
+
+    <!-- Sample Channel Type -->
+    <channel-type id="state">
+        <item-type>Number</item-type>
+        <label>Numeric state of the Indego</label>
+        <description>1 = mow, 2 = charging/docked, 3 = paused.</description>
+    </channel-type>
```

Comment:
I'd suggest to enumerate the possible states as options, i.e.

```xml
    <channel-type id="state">
         <item-type>Number</item-type>
         <label>State of the Indego</label>
         <description>1 = mow, 2 = charging/docked, 3 = paused.</description>
         <state readOnly="true">
            <options>
                <option value="1">mow</option>
                <option value="2">charging/docked</option>
                <option value="3">paused</option>
            </options>
         </state>
     </channel-type>    
```

That makes it much nicer to read within UIs